dalli.gemspec

```diff
@@ -15,6 +17,12 @@ Gem::Specification.new do |s|
     "Gemfile"
   ]
   s.homepage = "https://github.com/petergoldstein/dalli"
-  s.add_development_dependency "rack"
+  s.required_ruby_version = '>= 2.5'
+
   s.add_development_dependency "connection_pool"
+  s.add_development_dependency "rack"
+  s.add_development_dependency "rubocop"
+  s.add_development_dependency "rubocop-minitest"
+  s.add_development_dependency "rubocop-performance"
+  s.add_development_dependency "rubocop-rake"
```

Comment:
If all development is happening with Bundler, perhaps no development deps are necessary to add in the gemspec?

Reply:
Maybe not.  I don't have a strong opinion on that one way or the other.  Not sure it hurts.